php/src/Makefile.mk

```diff
@@ -10,7 +10,11 @@ IcePHP_targetdir        := lib
 IcePHP_installdir       := $(install_phplibdir)
 IcePHP_cppflags         := -I$(project) -I$(top_srcdir)/cpp/src $(ice_cpp_cppflags) $(php_cppflags)
 IcePHP_ldflags          := $(php_ldflags)
-IcePHP_dependencies     := IceDiscovery IceLocatorDiscovery IceSSL Ice
+IcePHP_dependencies     := IceDiscovery++11 IceLocatorDiscovery++11 IceSSL++11 Ice++11
+# TODO temporary ++11 dependencies are not linked
```

Comment:
What's the issue? Does this mean this PR doesn't work on macos?

Reply:
The issue is that ++11 libraries from dependencies are not automatically linked, and I have to add them to IcePHP_system_libs

Reply:
Would be nice if this temporary work-around worked on both Linux and macos.

Reply:
It does, CI is building fine with both.